Mathlib/MeasureTheory/Integral/Lebesgue.lean

```diff
@@ -800,6 +800,14 @@ theorem lintegral_indicator_one {s : Set α} (hs : MeasurableSet s) :
     ∫⁻ a, s.indicator 1 a ∂μ = μ s :=
   (lintegral_indicator_const hs _).trans <| one_mul _
 
+theorem measure_eq_measure_iff_lintegral_eq_lintegral (ν : Measure α) :
+    μ = ν ↔ ∀ f : α → ℝ≥0∞, Measurable f → ∫⁻ a, f a ∂μ = ∫⁻ a, f a ∂ν := by
```

Comment:
That's an extensionality lemma. It should be called `ext_iff_...`.
It would be nice to have the useful implication as a standalone lemma as well, called something like `ext_of_lintegral`.